composer.json

```diff
@@ -83,6 +83,8 @@
         "illuminate/view": "self.version"
     },
     "require-dev": {
+        "ext-gmp": "*",
+        "ext-bcmath": "*",
```

Comment:
Didn't we recently removed these? @timacdonald

Reply:
We were still referencing bcmath deps in some random tests. I've refactored those away now.